tests/device_tests/test_msg_cardano_get_address.py

```diff
@@ -16,17 +16,26 @@
 
 import pytest
 
-from trezorlib.cardano import PROTOCOL_MAGICS, get_address
-from trezorlib.tools import parse_path
+from trezorlib.cardano import (
+    NETWORK_IDS,
+    PROTOCOL_MAGICS,
+    create_address_parameters,
+    get_address,
+)
+from trezorlib.messages import CardanoAddressType
 
 from ..common import MNEMONIC12
 
+SHELLEY_TEST_VECTORS_MNEMONIC = (
+    "test walk nut penalty hip pave soap entry language right filter choice"
```

Comment:
are there published official vectors with this mnemonic?

if yes, please include links to them

if not, please regenerate vectors with "all all all..." * 12 mnemonic

Reply:
Done [here](https://github.com/trezor/trezor-firmware/pull/1112/commits/4f06bb81d7ef4a3d6fe59c1ded70f1f81907da00).